scripts/try_export_clowder_params.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Load Clowder params from json and export them as a environment variables.
+
+# Use go app command to print Clowder params
+function print_clowder_params() {
+  if [[ -n $GORUN ]]; then
+    go run main.go print_clowder_params
+  else
+    ./main print_clowder_params
+  fi
+}
+
+# Detect params, it should be printed in 'PARAM=VALUE' format
+CLOWDER_PARAMS=$(print_clowder_params | grep '=' || true) # use '|| true' not to stop on empty match
```

Comment:
`|| true` is redundant
`CLOWDER_PARAMS=$(print_clowder_params | grep '=' )` will not fail even if there's no `=` and script runs with `set -e`

Reply:
ok, taking back. it's correct